state/src/impls/test_helper.rs

```diff
@@ -457,7 +464,7 @@ macro_rules! set_top_level_state {
         set_top_level_state!($state, [$($x),*]);
     };
     ($state:expr, [(scheme: ($shard:expr, $addr:expr) => { amount: $amount:expr, metadata: $metadata:expr, approver: $approver:expr }) $(,$x:tt)*]) => {
-        assert_eq!(Ok((true)), $state.create_asset_scheme($shard, &$addr, $metadata, $amount, $approver, None, Vec::new()));
+        assert_eq!(Ok((true)), $state.create_asset_scheme($shard, &$addr, $metadata, $amount, $approver, None,Vec::new(), Vec::new()));
```

Comment:
It may be good to put space between `None,` and `Vec::new()`. I thought the formatter would do this...